src/components/Carousel.js

```diff
@@ -636,6 +638,9 @@ export default class Carousel extends Component {
    * @return {ReactElement} element
    */
   renderArrowRight = () => {
+    const { value, children, slides } = this.props;
+    const lastSlideIndex = slides ? slides.length - 1 : React.Children.count(children) - 1;
```

Comment:
```suggestion
    const lastSlideIndex = (slides ? slides.length : React.Children.count(children)) - 1;
```

Reply:
@piotr-s-brainhub committed the suggested change. Please check.

Thanks